client/base.jsx

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * Module dependencies.
+ */
+
+var React = require('react');
+
+module.exports = React.createClass({
+	render: function() {
+
+		var googleMapsScript =
+		    'function initialize() { dispatcher.dispatch(\'GOOGLE_MAPS_READY\'); } ' +
```

Comment:
Looks like we need some code conventions. You've been using tabs. I'd standardize on either that or 2-4 spaces.